pop-api/README.md

```diff
@@ -1 +1,73 @@
-Reserved crate for pop-api.
\ No newline at end of file
+# Pop API
+
+## Examples
+ink! smart contract examples using the Pop API
+- [balance-transfer](./examples/balance-transfer/)
+- [NFTs](./examples/nfts/)
+- [place-spot-order](./examples/place-spot-order/)
+- [read-runtime-state](./examples/read-runtime-state/)
+
+## What is the Pop API?
+
+One of the core value propositions of the Pop Network is to enable smart contracts to easily access the power of Polkadot. As such, the Pop API was built to enable smart contracts to easily utilize the functionality provided by the Pop Network parachain runtime.
+
+Substrate already exposes a Runtime API which is typically associated with the “outer node” calling into the runtime via RPC. Pop Network extends this concept in a direction that makes it usable by smart contracts, allowing untrusted contracts to make use of carefully exposed trusted functionality available within the runtime - this is the Pop API.
```

Comment:
I have my reservations here. Not sure whether they are comparable, and the relevance of comparing it with the "runtime api of the outer node."

Reply:
Still think that this whole bit is unnecessary. It is complicated and not really something a smart contract developer needs to understand.

Reply:
It is not special to pop api of pop network though. This is already possible since chain extensions were developed